app/clients/sms/twilio.py

```diff
@@ -78,6 +86,10 @@ def init_app(self, logger, callback_notify_url_host, environment, *args, **kwarg
             prefix = ""
 
         self.callback_url = f"https://{prefix}api.va.gov/vanotify/sms/deliverystatus"
+        self.callback_url += f"#ct={self._callback_connection_timeout}"
+        self.callback_url += f"&rc={self._callback_retry_count}"
+        self.callback_url += f"&rt={self._callback_read_timeout}"
+        self.callback_url += f"&rp={self._callback_retry_policy}"
```

Comment:
probably a slight performance hit here. Can the concatenation be done on one line with line breaks instead? something like
```
self.callback_url = f"https://{prefix}api.va.gov/vanotify/sms/deliverystatus" \
                    f"#ct={self._callback_connection_timeout}" \
                    f"&rc={self._callback_retry_count}" \
                    f"&rt={self._callback_read_timeout}" \
                    f"&rp={self._callback_retry_policy}"
```

Reply:
You're probably right. I broke it out to make a shorter line.